xml/security_pam.xml

```diff
@@ -728,7 +728,7 @@ DISPLAY     DEFAULT=${REMOTEHOST}:0.0  OVERRIDE=${DISPLAY}
      <para>
       To make sure the new authentication procedure works as planned, turn
       on debugging for all PAM-related operations. The <command>pam-config
-      --add --ldap-debug</command> turns on debugging for LDAP-related PAM
+      --add --sss-debug</command> turns on debugging for SSSD-related PAM
```

Comment:
This isn't how sssd debugging works. the --sss-debug option just adds debug to the pam to resolver interactions, it's not changing the debug log level in sssd itself. As a result, this section needs to be updated to reflect how to debug sssd. 

Additionally, is there any documentation here about configuring sssd.conf? 

Could someone realistically follow this chapter from start to finish and be able to log in to a system with ldap authentication? That's the barrier we have to clear.

Reply:
Is this what you are referring to, as an example:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/configuring-sssd-to-use-ldap-and-require-tls-authentication_configuring-authentication-and-authorization-in-rhel
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sssd-troubleshooting

Reply:
The context of this bz was to replace ldap with sssd , but yes that could be added as part of this .

Reply:
Or link to a location where we actually show the full setup of how to configure sssd.

Reply:
I can add a section on configuring SSSD , Will tag you once I ve it ready